recipes/catch2/2.x.x/test_package/300-benchmark.cpp

```diff
@@ -0,0 +1,15 @@
+#define CATCH_CONFIG_ENABLE_BENCHMARKING
```

Comment:
Don't add a define here.
We want to test whether Conan correctly passes this define.